### ISSUE TITLE
Loss Function

### ISSUE THREAD

**Issue by weilingwei96** [Luoyadan/BGNN-AAAI#3]
Hi, 
I see that the proposed model is optimized by minimizing the binary cross-entropy loss of query edges and Bayes by Backprop loss. But I failed to locate the calculation of Bayes loss L_B in the code. 

Can you explain for me? 
Thank you.

**Comment by Luoyadan** [comment]
Hi,

The Bayes by Backprop can be found in utils.py. It is inspired by https://github.com/Gordonjo/versa/blob/master/src/utilities.py.

Cheers,
Yadan

**Comment by Wolfybox** [comment]
Bayesian loss is not explicitly implemented as it is entailed in the cross entropy loss.
